vnext/Microsoft.ReactNative/ReactNativeHost.idl

```diff
@@ -4,7 +4,7 @@
 import "IReactPackageProvider.idl";
 import "ReactInstanceManager.idl";
 import "ReactInstanceSettings.idl";
-import "ReactRootView.idl";
+import "ReactRootView2.idl";
```

Comment:
>ReactRootView2 [](start = 8, length = 14)

Is there a separate task to rename it later?

Reply:
There is no task, but it is part of my current work to improve the instance management. 

---
In reply to: [360532579](https://github.com/microsoft/react-native-windows/pull/3809#discussion_r360532579) [](ancestors = 360532579)